core/cog-launcher.c

```diff
@@ -346,7 +378,6 @@ cookie_manager_get_accept_policy (WebKitCookieManager *cookie_manager)
     return data.result;
 }
 
-
```

Comment:
Please remove this bit from the patch, it's only adding noise to the diff.

Reply:
This line will make noise all the time... anyway.